plugins/renamerOnUpdate/renamerOnUpdate.py

```diff
@@ -486,7 +489,12 @@ def exit_plugin(msg=None, err=None):
     new_filename = re.sub("[’‘”“]+", "'", new_filename)
 
 # Replace the old filename by the new in the filepath
-new_path = current_path.rstrip(current_filename) + new_filename
+#log.LogWarning("Old path {}".format(current_path))
```

Comment:
I don't know if leaving this commented out logging in is a problem or not.  I kept having to turn it on during testing to see how I was mishandling paths.